pkg/controller/v1beta1/inferenceservice/controller.go

```diff
@@ -369,3 +378,19 @@ func (r *InferenceServiceReconciler) deleteExternalResources(isvc *v1beta1api.In
 	}
 	return nil
 }
+
+func (r *InferenceServiceReconciler) GetFailConditions(isvc *v1beta1api.InferenceService) string {
+	/*
+		logf.SetLogger(zap.New())
+		log := logf.Log.WithName("my_controller")
+		log.Info("GetFailConditions")
+		fmt.Println("my_GetFailConditions")
```

Comment:
left over?

Reply:
Yes, it's a missed deleted part of the draft. We'll take care of this, thanks!